santropolroulant/sous-chef
# issue #52 Specify upcoming dates where the client has chosen not to receive any meal

To test :

1- Choose a client
2- Edit the client and go to the tab preferences
3- Make sure that the day you want to generate the order for is in shedule
![image](https://user-images.githubusercontent.com/4096656/106367182-c58d5d00-630e-11eb-992e-ccd5daf5f86b.png)
4- Select an upcoming date that you want to cancel the generation of the order (Today) and save.
![image](https://user-images.githubusercontent.com/4096656/106367213-f1104780-630e-11eb-8c50-1bf63904c919.png)
5-Go to Kitchen Count and click generate orders, the order will not be generated for this particular customer.
